README.md

```diff
@@ -29,31 +30,36 @@ Inspired by [ziadoz/awesome-php](https://github.com/ziadoz/awesome-php)
 - [Miscellaneous](#miscellaneous)
 
 ## Essentials
-* [Laravel](http://laravel.com)
-* [Laravel Documentation](http://laravel.com/docs)
-* [Laravel API Reference](http://laravel.com/api/master)
-* [Lumen](http://lumen.laravel.com)
-* [Lumen Documentation](http://lumen.laravel.com/docs)
-* [Laracasts](http://laracasts.com)
+
+* [Laravel](https://laravel.com)
+* [Laravel Documentation](https://laravel.com/docs/5.3)
+* [Laravel API Reference](https://laravel.com/api/master/)
+* [Lumen](https://lumen.laravel.com)
+* [Lumen Documentation](https://lumen.laravel.com/docs/5.2)
```

Comment:
Lets keep it like this for now and update it in the future. We're only required to update the version number once or twice a year.